.github/workflows/insitu.yml

```
@@ -30,3 +30,23 @@ jobs:
     - name: Build
       run: |
         cmake --build build -j 2
+  ascent:
+    name: Ascent
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.draft == false
+    env:
+      CXX: g++
+      CC: gcc
+      CMAKE_PREFIX_PATH: /home/user/ascent/install-debug/lib/cmake/
+    container:
+      image: alpinedav/ascent:ascent-ubuntu-18_2021-10-23-shaba7872
+    steps:
+    - uses: actions/checkout@v2
+    - name: Configure
+      run: |
+        . ../s_env.sh && cmake -S . -B build               \
+                                    -DWarpX_ASCENT=ON      \
+                                    -DWarpX_COMPUTE=NOACC
+    - name: Build
+      run: |
+        . ../s_env.sh && cmake --build build -j 2
```

Just an end-of-file newline:
```suggestion
        . ../s_env.sh && cmake --build build -j 2
```

---

I can't commit this one for some reason.

My first pass I had an end-of-file nl, but i failed a style check so I removed it :-)

---

You can just not add empty spaces at the end of a line in our style ^^
Empty newlines are ok :)